securedrop_client/app.py

```diff
@@ -45,7 +45,7 @@
 
 DEFAULT_LANGUAGE = LanguageCode("en")
 DEFAULT_SDC_HOME = "~/.securedrop_client"
-DESKTOP_FILE_NAME = "org.freedomofthepress.securedrop.client"
+DESKTOP_FILE_NAME = "press.freedom.securedrop.client"
```

Comment:
```suggestion
DESKTOP_FILE_NAME = "press.freedom.SecureDropClient.desktop"
```

This is one way we could implement the _freedesktop_ guidelines. ([specifications](https://specifications.freedesktop.org/desktop-entry-spec/latest/ar01s02.html))